### PR TITLE
RSS Feed / OPML reader and article parser

### PR DESCRIPTION
**Pull request by ruze00**
Submitted to llama-index and tested.  PR here also in response to suggestion by @jerryjliu.

### CODE REVIEW

**Comment by jerryjliu** [comment]
awesome thanks! will review soon

File: llama_hub/web/news/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# News Article Loader
+
+This loader makes use of the `newspaper3k` library to parse web page urls which have news
+articles in them.
+
+## Usage
+Pass in an array of individual page URLs:
+```python
+from llama_index import download_loader
```
Comment (jerryjliu) [inline llama_hub/web/news/README.md:9]: canonical way to use loaders is now `from llama_hub.web.news.base import NewsArticleReader`

File: llama_hub/web/news/base.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            if self.use_nlp:
+                metadata["keywords"] = getattr(article, "keywords", [])
+                metadata["summary"] = getattr(article, "summary", "")
+
+            documents.append(Document(text=content, metadata=metadata))
+
+        return documents
+
+
+class RssNewsReader(BaseReader):
```
Comment (jerryjliu) [inline llama_hub/web/news/base.py:91]: could you make this a separate reader with its own folder? will keep every entry distinct

**Review by jerryjliu — COMMENTED** [review]
thanks - once you separate out both loaders, would you be able to add both to library.json?

File: llama_hub/web/news/base.py (hunk 2 above)
Comment (ruze00) [reply llama_hub/web/news/base.py:91]: Sure.  There is already an `rss` folder.  Should I make it `rss_news` or other preference?

Comment (ruze00) [reply llama_hub/web/news/base.py]: updated in the meanwhile.

File: llama_hub/library.json [hunk 3]
```diff
@@ -622,5 +622,23 @@
       "bucket",
       "storage"
     ]
+  },
+  "NewsArticleReader": {
+    "id": "news",
```
Comment (jerryjliu) [inline llama_hub/library.json:627]: id needs to be `web/news` and `web/rss_news` (to show up on llamahub UI)

**Review by jerryjliu — APPROVED** [review]
awesome! one final comment but approving to unblock

File: llama_hub/library.json (hunk 3 above)
Comment (danipolo) [reply llama_hub/library.json]: this seems to be broken

**Pull request merged**